layouts/_default/baseof.html

```diff
@@ -9,7 +9,7 @@
 
   {{- partial "global_navbar.html" . -}}
   {{ $path := split $.Source.File.Path .Site.Params.PathSeparator }}
-  {{ if or (eq (index $path 0) "events") (eq .Type "events") }}
```

Comment:
As I don't actually parse what you're saying here, I'll wait for you to test. See my tests in https://github.com/devopsdays/devopsdays-theme/issues/468 to see where my thinking is.